docs/SUMMARY.md

```diff
@@ -3,27 +3,28 @@
-		* [Supported storage systems](content/release_notes/supported_storage.md)
-		* [Supported operating systems](content/release_notes/supported_os.md)
-		* [Supported orchestration platforms](content/release_notes/supported_orchestration.md)
-	* [Change log](content/release_notes/change_log.md)
-		* [1.11.0 (July 2022)](content/release_notes/changelog_1.11.0.md)
-		* [1.10.0 (July 2022)](content/release_notes/changelog_1.10.0.md)
-		* [1.9.0 (March 2022)](content/release_notes/changelog_1.9.0.md)
-		* [1.8.0 (December 2021)](content/release_notes/changelog_1.8.0.md)
-		* [1.7.0 (September 2021)](content/release_notes/changelog_1.7.0.md)
-		* [1.6.0 (June 2021)](content/release_notes/changelog_1.6.0.md)
-		* [1.5.1 (July 2021)](content/release_notes/changelog_1.5.1.md)
-		* [1.5.0 (March 2021)](content/release_notes/changelog_1.5.0.md)
-		* [1.4.0 (December 2020)](content/release_notes/changelog_1.4.0.md)
-		* [1.3.0 (September 2020)](content/release_notes/changelog_1.3.0.md)
-		* [1.2.0 (June 2020)](content/release_notes/changelog_1.2.0.md)
-		* [1.1.0 (March 2020)](content/release_notes/changelog_1.1.0.md)
-		* [1.0.0 (November 2019)](content/release_notes/changelog_1.0.0.md)
-	* [Limitations](content/release_notes/limitations.md)
-	* [Known issues](content/release_notes/known_issues.md)
+    * [What's new in 1.11.0](content/release_notes/whats_new.md)
+    * [Compatibility and requirements](content/release_notes/compatibility_requirements.md)
+        * [Supported storage systems](content/release_notes/supported_storage.md)
+        * [Supported operating systems](content/release_notes/supported_os.md)
+        * [Supported orchestration platforms](content/release_notes/supported_orchestration.md)
+    * [Change log](content/release_notes/change_log.md)
+        * [1.11.1 (May 2023)](content/release_notes/changelog_1.11.1.md)
+        * [1.11.0 (January 2023)](content/release_notes/changelog_1.11.0.md)
+        * [1.10.0 (July 2022)](content/release_notes/changelog_1.10.0.md)
+        * [1.9.0 (March 2022)](content/release_notes/changelog_1.9.0.md)
+        * [1.8.0 (December 2021)](content/release_notes/changelog_1.8.0.md)
+        * [1.7.0 (September 2021)](content/release_notes/changelog_1.7.0.md)
+        * [1.6.0 (June 2021)](content/release_notes/changelog_1.6.0.md)
+        * [1.5.1 (July 2021)](content/release_notes/changelog_1.5.1.md)
+        * [1.5.0 (March 2021)](content/release_notes/changelog_1.5.0.md)
+        * [1.4.0 (December 2020)](content/release_notes/changelog_1.4.0.md)
+        * [1.3.0 (September 2020)](content/release_notes/changelog_1.3.0.md)
+        * [1.2.0 (June 2020)](content/release_notes/changelog_1.2.0.md)
+        * [1.1.0 (March 2020)](content/release_notes/changelog_1.1.0.md)
+        * [1.0.0 (November 2019)](content/release_notes/changelog_1.0.0.md)
+    * [Limitations](content/release_notes/limitations.md)
+    * [Known issues](content/release_notes/known_issues.md)
```

Comment:
it seems that there is indentation change here, any reason?

Reply:
Yes, I fixed a indentation problem from the previous version

Reply:
fixed